src/tools/vmap4_extractor/vmapexport.cpp

```diff
@@ -488,8 +488,8 @@ int main(int argc, char ** argv)
 #endif
                     ))
             success = (errno == EEXIST);
-
-    int FirstLocale = -1;
+    int defaultFirstLocale = -1;
```

Comment:
From discussion @ irc, @Shauren suggested to not even declare that default and instead just use -1 straight up here and in the test.